api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionEmailAddress.java

```diff
@@ -0,0 +1,71 @@
+package org.pmiops.workbench.db.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "institution_email_address")
+public class DbInstitutionEmailAddress {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "institution_email_address_id")
+  private long institutionEmailAddressId;
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "institution_id")
+  private DbInstitution institution;
+
+  @Column(name = "email_address", nullable = false)
+  private String emailAddress;
+
+  public DbInstitutionEmailAddress() {}
+
+  public DbInstitutionEmailAddress(DbInstitution institution, String emailAddress) {
+    this.institution = institution;
+    this.emailAddress = emailAddress;
+  }
+
+  public DbInstitution getInstitution() {
+    return institution;
+  }
+
+  public String getEmailAddress() {
+    return emailAddress;
+  }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
generated these because we use them in Sets

Reply:
Nice, i typically always implement equals/hashcode for all jpa model objects, but sometimes object equality is enough.